File: array/span.java
import java.util.Scanner;

public class span {
    // headline =
    // step 1:code Dragen step by step
    // step 2: take a input from user 0 to arr.length;
    // step 3: make two variable on is min and one is max
    // step 4: first when i check loop is start 0 to n check arr[i]>max;
    // inside a loop is call arr[i]<min;
    public static void main(String[] args) {

        System.out.println("Enter a number :");
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int arr[] = new int[n];

        for (int i = 0; i < arr.length; i++) {
            arr[i] = sc.nextInt();
        }
        int max = arr[0];
        int min = arr[0];

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] > max) {
                max = arr[i];
            }
            if (arr[i] < min) {
                min = arr[i];

            }
        }
        int span = max - min;
        System.out.println("This is span Number: " + span);

    }

}

File: array/arrdart/span.dart
import 'dart:io';

sapanArray (int arr){

// step 1 first take a Input from user and 
// step 2 apply the loop 1 to array.length;
// step 3 every time check write a condition arr[i]>max 
// step 4 if arr[i] is max so max = arr[i];
// step 5 reapet and reapt again if array is max check second Index 
// step 6 same check if arr[i]<min;
// step 7 check every time all  of the loop 
stdout.write("Enter  a number:");
List<int> arr = [];
int max= arr[0];
int min =arr[0];

for(int i =0;i<arr.length;i++){
  if(arr[i]>max){
    max= arr[i];
  }
  if(arr[i]<min){
    min=arr[i];
  }
}
}
void main(){
  sapanArray(10);
}